sys-apps/systemd/systemd-9999.ebuild

```diff
@@ -558,11 +561,19 @@ pkg_postinst() {
 	# Flatcar: Reenabling systemd-timesyncd service too.
 	systemd_reenable systemd-networkd.service systemd-resolved.service systemd-timesyncd.service
```

Comment:
It's a bit longer ago that I touched this, what is the difference between `systemd_reenable` and `save_enabled_units`?

Reply:
Maybe add `getty@tty1.service` here?

Reply:
"reenable" is "disable" + "enable". `save_enabled_units` is supposed to remember which of the listed units were remembered in preinst step, so in postinst, we enable them again. But the enabling means putting stuff to `/etc`, since it's done with `systemctl enable|reenable`. We don't want that. So I guess we should rewrite those, or make sure that they are not actually called?

Reply:
Reenable is used for `systemd-networkd.service systemd-resolved.service systemd-timesyncd.service` and all are correctly placed under `/usr`.
I don't know if it has any unwanted side effect if we also used if for `machines.target` or `remote-*.target`. It seems both `save_enabled_units` and `systemd_reenable` only work if the units were enabled – I guess as long as we have the result we want it's ok to use both as needed.

Reply:
This line got dropped, since it was basically a noop.